.github/workflows/format-pr.yml

```diff
@@ -45,13 +45,13 @@ jobs:
           node-version: 18.x
 
       - name: Install dependencies
-        run: npx @microsoft/rush update
```

Comment:
@mikeharder for my context is this more reliable because we avoid npx installing and pulling down this package every time?

Reply:
Yes, it's slightly faster and more reliable because it avoids downloading the latest rush package every time.

`rush` has a "version selector" built in to the tool, so the version of `rush` initially run from the command line is just a shim that loads and runs the version set here: 

https://github.com/microsoft/typespec/blob/18f85c828b94f76db7df6c66a5ca89cc47ceaab3/rush.json#L7

So as long as the "version selector" is not broken in the latest version of `rush` on npmjs.com, it should be pretty safe to use `npx @microsoft/rush`.  But there is still a small risk that the "version selector" does break (it happened once but was quickly fixed by the rush team).

Running the script checked into the repo is guaranteed to never break, and is also a little faster since we only need to install the version of `rush` in `rush.json`, rather than installing two versions (latest and specified).  The scripts checked into the repo are automatically upgraded by `rush`, when we upgrade `rush` to a version that includes script changes.